### PR TITLE
[WIP] Action to render output to .wav

### PR DESCRIPTION
**Pull request by vaibhavsagar**
Still need to figure out how to hook up the GitHub Action for making a new release (unless we're putting the generated .wav somewhere else).

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: "Build"
+on:
+  pull_request:
+  push:
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (soenkehahn) [inline .github/workflows/build.yml]: Can we pin down the ubuntu version here? To make this more reproducible?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        sudo apt-get update
+        sudo apt-get install --yes curl libsndfile1-dev portaudio19-dev
+    - uses: actions/cache@v1
+      with:
+        path: ~/.stack
+        key: ${{ runner.os }}-stack
+    - uses: actions/checkout@v2
+      with:
+        repository: 'soenkehahn/looper'
+        path: 'looper'
```
Comment (soenkehahn) [inline .github/workflows/build.yml:19]: Can we pin the checkout of looper to the current commit?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    - run: |
+        cd looper
+        stack install
+        cd ..
+    - uses: actions/checkout@v2
+    - run: |
+        export PATH="$HOME/.local/bin:$PATH"
+        looper run.sh --render output.wav
```
Comment (soenkehahn) [inline .github/workflows/build.yml]: I think this file should be named `rc-loops.wav`.

**Review by soenkehahn — COMMENTED** [review]
This is great, thanks a lot.

todo: Make the output file accessible for download publicly. E.g. through github artifacts.

File: .github/workflows/build.yml (hunk 1 above)
Comment (vaibhavsagar) [reply .github/workflows/build.yml]: Sure, updated.

File: .github/workflows/build.yml (hunk 2 above)
Comment (vaibhavsagar) [reply .github/workflows/build.yml:19]: Done.

File: .github/workflows/build.yml (hunk 3 above)
Comment (vaibhavsagar) [reply .github/workflows/build.yml]: Good idea, updated.

**Pull request closed without merging**